ca_gazebo/launch/create_two_floors_house.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="virtual_wall_x" default="0.2"/>
   <arg name="virtual_wall_y" default="-1.2"/>
-  <arg name="virtual_wall_yaw" default="1.57"/>
+  <arg name="virtual_wall_yaw" default="-1.57"/>
```

Comment:
Why did you do this?

Reply:
To "point" out the virtual_wall to detect it in-front of the entrance, the original position detects on the opposite direction

Reply:
Would you send a screenshot showing why this was necessary? I can't see the benefit.

Reply:
![two_f1](https://user-images.githubusercontent.com/53347092/70150135-3fcd3d80-1688-11ea-91ca-05dc88c90301.png)

Originally the virtual wall laser points in that direction, but there is also wall there, so is redundant to the system since it have to avoid the walls anyway.

So rotating 180 degrees makes it point to the direction where is an actual opening that the robot could pass if the virtual wall is not there, making it better for testing his behavior